andela/ah-frontend-poseidon
# #162162970 Pagination support for articles

**What does this PR do?**
- Implement previous/next page buttons
- Fix the details view for a single article without Images
- Adjust the spacing between **Edit**, **Delete** buttons and **tags**.
- Centralize the details view for a single article
- Write tests

**Description of the tasks to be completed?**
- Add previous, current, and next page buttons on the home component.
- Write a function to fetch articles
- Add css to fix `single article page`
- Write tests

**How to manually test it**
- Clone the repository by running `git clone https://github.com/andela/ah-frontend-poseidon.git`
- Run `git checkout ft-create-articles-162162967`
- Start the server by running `yarn start`
- Click `Previous` or `Next` button when the page finishes loading and you should see different articles and the circle button in the middle changing.
- Click one of the `read more` buttons to view one article

**Screenshots**
<img width="1204" alt="screen shot 2019-01-30 at 5 20 59 pm" src="https://user-images.githubusercontent.com/36928255/51987505-b2dbb580-24b3-11e9-8c04-f3258b2fb3cf.png">
<img width="1440" alt="screen shot 2019-01-29 at 8 17 59 pm" src="https://user-images.githubusercontent.com/36928255/51926754-0b9f4580-2403-11e9-8ec4-198422e4a905.png">



**What are the relevant pivotal tracker stories?**
#162162970
